eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/immutable/AbstractImmutableBiMap.java

```diff
@@ -331,6 +333,26 @@ public ImmutableShortBag collectShort(ShortFunction<? super V> shortFunction)
         return this.delegate.zipWithIndex(new UnifiedSet<Pair<V, Integer>>()).toImmutable();
     }
 
+    public <V1> ImmutableObjectLongMap<V1> sumByInt(Function<? super V, ? extends V1> groupBy, IntFunction<? super V> function)
+    {
+        return this.getDelegate().sumByInt(groupBy, function).toImmutable();
+    }
+
+    public <V1> ImmutableObjectDoubleMap<V1> sumByFloat(Function<? super V, ? extends V1> groupBy, FloatFunction<? super V> function)
+    {
+        return this.getDelegate().sumByFloat(groupBy, function).toImmutable();
+    }
+
+    public <V1> ImmutableObjectLongMap<V1> sumByLong(Function<? super V, ? extends V1> groupBy, LongFunction<? super V> function)
+    {
+        return this.getDelegate().sumByLong(groupBy, function).toImmutable();
+    }
+
+    public <V1> ImmutableObjectDoubleMap<V1> sumByDouble(Function<? super V, ? extends V1> groupBy, DoubleFunction<? super V> function)
+    {
+        return this.getDelegate().sumByDouble(groupBy, function).toImmutable();
+    }
+
```

Comment:
You can call `this.delegate` instead of `this.getDelegate()` in these 4 methods.